### PR TITLE
Implemented "not logged in" screens + added menu buttons

### LINKED ISSUE #46: Implement "Not logged in" screens.
Inbox

### LINKED ISSUE #47: Implement "Menu" component
Menu

### PR DESCRIPTION
**Pull request by lcflint**
closes #46
closes #47

### CODE REVIEW

**Comment by jsindos** [comment]
Hey Lawrence, looks really good!

I really like your naming of the components, for example you condensed “Landing Join POP Scrollable Menu” to “LandingJoinScrollable” which I think is a really good, the first is too verbose.

There are a few minor things:

## Menu

- Few spacing issues
- Isn't using the same colour as the design (should be #EB5757)
- I don't think it's using the right font (OpenSans-Regular and OpenSans-Bold)
- The selected menu item should be a bolded font and #222 in colour

For comparison:

image

vs

image

When I navigate to Bag, Inbox and Profile I get the following error on mobile, I think it's probably because some text isn't wrapped inside a <Text> tag.

IMG_0406

When I run the app on my iPhone 8 and scroll to the bottom, the bottom is cut off (I can't scroll any further). Probably just needs a little more padding on the bottom of the Join Pop Scrollable.

IMG_0405

I saw your MenuItem btw

```
// TODO: Move to it's own component file?
const MenuItem = (props) =>
  <TouchableOpacity style={styles.menuItem}>
    <Text style={styles.menuItemText}>{props.text}</Text>
    <Image source={props.image} style={styles.menuItemImage}/>
  </TouchableOpacity>
```

We'll talk about this in the meeting but basically I think all re-usable components should be moved to a ReusableComponents top-level directory. These are different to styles/sharedComponents.js which are styled components and have very little functionality. Based on this I think we should also re-name sharedComponents.js to sharedStyledComponents to make it more explicit (will talk about this in the meeting).

Basically most components from Figma will live in the ReusableComponents directory.

I also think we should follow this wiki page I chucked in for directory structure in the screens directory: https://github.com/jsindos/pop/wiki/Directory-Structure

Let me know if you have any questions, will talk more about it with you in meeting.
